### PR TITLE
Create a new SpectatorView.Example.Unity project to avoid external dependencies being declared in SpectatorView.Unity

### PR DESCRIPTION
**Pull request by chrisfromwork**

### CODE REVIEW

File: external/MixedReality-QRCodePlugin/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*
```
Comment (andreiborodin) [inline external/MixedReality-QRCodePlugin/.gitignore]: Why are these files here?

**Comment by andreiborodin** [comment]
>   assetBundleVariant: 

Why was this folder moved out from SpectatorView.Unity? It seems integral to SpectatorView.Unity working with ARKit?

---
Refers to: src/SpectatorView.Unity/Assets/ARKit.meta:8 in 78cc43e. [](commit_id = 78cc43ed0666c6d326d0109f47c0e42b8f473544, deletion_comment = True)

File: external/MixedReality-QRCodePlugin/.gitignore (hunk 1 above)
Comment (chrisfromwork) [reply external/MixedReality-QRCodePlugin/.gitignore]: Chatted offline, adding comments
